libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/AudioFocusDelegate.kt

```diff
@@ -1,8 +1,18 @@
 package com.mapbox.navigation.ui.voice.api
 
-internal interface AudioFocusDelegate {
+/**
+ * An Api that allows you to interact with the audio focus
+ */
+interface AudioFocusDelegate {
 
-    fun requestFocus()
+    /**
+     * Request audio focus. Send a request to obtain the audio focus
+     * @return true on successful focus change request.
+     */
+    fun requestFocus(): Boolean
 
+    /**
+     * Abandon audio focus. Causes the previous focus owner, if any, to receive focus.
+     */
     fun abandonFocus()
```

Comment:
Similar to above comment. Wondering if we should return the different return values (as Android does) 🤔 

```java
*  @return {@link #AUDIOFOCUS_REQUEST_FAILED} or {@link #AUDIOFOCUS_REQUEST_GRANTED}
```

Reply:
No idea how to handle result and do we need to handle it?